config/groups.yaml

```diff
@@ -720,3 +720,10 @@
     - "2695849"
     - "2683894"
   id: 190130
+  - name: "NEMP"
+  members:
```

Comment:
This isn't valid YAML: the indentation matters. You'll need to fix this before we can proceed. See the other entries in the file for how to format this.